packages/core/src/definitions/styles.ts

```diff
@@ -96,7 +96,7 @@ export const builtInStyles: Partial<
     type: 'Text',
     group: 'style',
     description: 'The orientation of the section',
-    defaultValue: 'row',
+    defaultValue: 'column',
```

Comment:
This value changed to `'column'` in the sdk
- Same line in the file being deleted: [variables.ts#L102](https://github.com/contentful/experience-builder/pull/214/files#diff-5391ff296f3c429f92a2d0db7ace344511b03a37567e20c9687ce439abfa4b1fL102)